ibm/resource_ibm_iam_access_group_members.go

```diff
@@ -300,7 +308,9 @@ func resourceIBMIAMAccessGroupMembersDelete(context context.Context, d *schema.R
 		if err != nil {
 			return diag.FromErr(err)
 		}
-		err = iamuumClient.AccessGroupMember().Remove(grpID, ibmUniqueID)
+
+		removeMembersFromAccessGroupOptions := iamAccessGroupsClient.NewRemoveMemberFromAccessGroupOptions(grpID, ibmUniqueID)
+		_, err = iamAccessGroupsClient.RemoveMemberFromAccessGroup(removeMembersFromAccessGroupOptions)
 		if err != nil {
 			return diag.FromErr(err)
```

Comment:
return diag.FromErr(err) ===> return diag.FromErr(fmt.Errorf("Error removing members to group(%s). API response: %s", grpID, detailResponse))